development_deploy.sh

```diff
@@ -0,0 +1,8 @@
+npm i -g surge
+
+export SURGE_LOGIN=test@example.co.in
+# Token of a dummy account.
+export SURGE_TOKEN=d1c28a7a75967cc2b4c852cca0d12206
```

Comment:
This is probably a bad idea. These tokens (new generated ones) should go into the travis environment variables on travis' CI and shouldn't be publicly exposed in any of the files. You need to `encrypt` these tokens before they're merged to maintain security.

Reply:
The surge tokens were already used in this project previously. I am not sure about encrypting though. It is publicly available on fossasia's blog post(https://blog.fossasia.org/installing-the-loklak-search-and-deploying-it-to-surge). There might be a method to not to share this token, but it requires access to repo. Or someone with right access would have to push travis yml configuration on respective travis account. For more info please see: https://docs.travis-ci.com/user/deployment/surge/ 

Please guide me further.

Reply:
@sudheesh001 please review.